### PR TITLE
fix(date): Change the date format to HH:MM:SS in order to be consiste…

### PR DESCRIPTION
**Pull request by ioanpocol**
…nt over the application

### CODE REVIEW

**Comment by petrjasek** [comment]
:+1:

File: features/steps/template_steps.py [hunk 1]
```diff
@@ -21,5 +21,5 @@ def then_next_run_is_on_monday(context, time):
     next_run = parse_date(data.get('next_run'))
     assert isinstance(next_run, datetime)
     assert next_run.weekday() == 0
-    assert next_run.strftime('%H%M') == time
+    assert next_run.strftime('%H:%M:%S') == time
     assert next_run.strftime('%S') == '00', 'there should be no seconds, but it was %s' % (next_run.strftime('%S'), )
```
Comment (naomiaro) [inline features/steps/template_steps.py]: prob need to get rid of this assert now for no seconds.

File: superdesk/utc.py [hunk 2]
```diff
@@ -76,14 +76,14 @@ def utc_to_local(local_tz_name, utc_datetime):
...
     :param datetime current_datetime
     :param string timestr
     :param int second
     """
     if timestr is None:
-        timestr = '0000'
-    time = datetime.datetime.strptime(timestr, '%H%M')
+        timestr = '00:00:00'
+    time = datetime.datetime.strptime(timestr, '%H:%M:%S')
     return current_datetime.replace(hour=time.hour, minute=time.minute, second=second)
```
Comment (naomiaro) [inline superdesk/utc.py]: should use time.second here as well, now that the time can have seconds.

File: superdesk/utc.py [hunk 3]
```diff
@@ -76,14 +76,14 @@ def utc_to_local(local_tz_name, utc_datetime):
 def set_time(current_datetime, timestr, second=0):
```
Comment (naomiaro) [inline superdesk/utc.py]: don't need this second=0

**Comment by naomiaro** [comment]
There's a few time things I've posted on a couple lines.

**Pull request merged**